### PR TITLE
™The Tour - Kevin's Edition version 1.0

### PR DESCRIPTION
**Pull request by gunr2171**
I'm creating the pull request because I do think this is a good idea, but I have some comments that I want to keep track of.

### CODE REVIEW

File: _posts/2016-06-02-tour.md [hunk 1]
```diff
@@ -3,19 +3,58 @@ layout: default
...
+    [tag:cv-pls] The reason of why you're voting to close the question <link-to-the-question>
+
+For example:
+
+![Good cv-pls request example image](https://i.stack.imgur.com/MXDfY.png)
+
+<sub>Tip: We have [an user-script to generates the `[cv-pls]` requests automatically](https://github.com/SO-Close-Vote-Reviewers/UserScripts#so-close-vote-request-generator).</sub>
+
+
+Everyone can post `[cv-pls]` requests in the chat room when you think a question should be closed quickly. Here's the rules about the request - **always remember and follow them**:
```
Comment (gunr2171) [inline _posts/2016-06-02-tour.md:29]: > **always remember and follow them**

Might want to be less assertive here, or not have anything.

File: _posts/2016-06-02-tour.md [hunk 2]
```diff
@@ -3,19 +3,58 @@ layout: default
...
 
-## ALLOWED
 
- - You can post a [\*-pls] request, *but* please use the following format: "[\*-pls] **REASON** post link" (\*close/reopen/delete/undelete)
-   - These requests should NOT be a habit and only be used when needed
-   - Please don't spam the room with requests
+# What SOCVR is
 
-## NOT ALLOWED
+The [SO Close Vote Reviewers](http://chat.stackoverflow.com/rooms/info/41570/so-close-vote-reviewers) is a team, the main goal of ours is that **close the bad questions as quickly as possible**.
```
Comment (gunr2171) [inline _posts/2016-06-02-tour.md:13]: Technically, the primary goal is not to close things _quickly_, just to close stuff that needs to be closed. It used to be that we focused just on the CV queue, but lately we've expanded on CV requests and bot-assisted reviews.

**Comment by gunr2171** [comment]
Idea: make the whole cv-pls section briefer. We can merge all the rules and restrictions into a "don't abuse the system and use your best judgement when selecting posts to send to chat. Read more at [link to full faq cv-pls section].

**Comment by mogsdad** [comment]
Rather than critique the content of this request, my concern is about whether it meets the goal of a "Tour" section. Consider [the SO tour](http://stackoverflow.com/tour), which introduces the purpose and standards for the site, but also guides new users through just 6 common first activities that are core to using the site. Our tour could (should!) be something similar, introducing prospective new members to the moderating activities & community in just a few steps.

For example, we have details about how to use cv-pls in the FAQ, so a repeat of that information in the "tour" should add some new value. What I would expect, then, would be an example of producing one.

**Comment by K-Guan** [comment]
Ah, okay I understand. Our Tour should be _guiding people about how to join SOCVR quickly_. Not a short version of the FAQ.

Let me edit the Tour and make it clear and simple - save the really important stuff, and remove these useless parts.

**Comment by mogsdad** [comment]
If it helps, I made an experimental start here: https://github.com/SO-Close-Vote-Reviewers/SO-Close-Vote-Reviewers.github.io/blob/mogdad-dev/_posts/2016-06-02-tour.md

It has the sort of feel I think will engage people, and quickly introduce them to SOCVR.

**Comment by K-Guan** [comment]
@mogsdad Oh, that looks great. Maybe we just need to explain what is `[cv-pls]` (also with an image) instead of the **ALLOWED** stuff if needed and then it's enough I think. Not just a start, but a complete tour.

@gunr2171 So, what do you think about Mogsdad's version?

**Comment by mogsdad** [comment]
Re: cv-pls, I think you're right - a picture of a cv-pls request in the
edit box (with formatting) would be good. Text along the lines of, "[In
some cases](link to cv-pls FAQ), you can request that others on the team
review a particular question.

Before that, though, we should introduce "the queue", and close-events...
they are the reason the room came into existence in the first place.

That would be a great start. Eventually, we could expand it to introduce
other curation activities, but if we cover those basics, I think the "tour"
will be established.

**Pull request closed without merging**